src/reducers/players.js

```diff
@@ -83,6 +84,25 @@ const players = (state = initialState, action) => {
         },
       });
     }
+    case SHUFFLE_CARD: {
+      // formula from MDN => Math.floor(Math.random() * (max - min + 1)) + min ... here, min = 0
```

Comment:
Make a new file `src/utils.js` and add the formula there and then import it to this file